pytorch_pfn_extras/onnx/pfto_exporter/export.py

```diff
@@ -168,6 +168,14 @@ def _type_to_proto(t: torch._C.TensorType) -> onnx.TypeProto:
 
     if t.scalarType() is None:
         ret.tensor_type.elem_type = onnx.TensorProto.DataType.UNDEFINED  # type: ignore[attr-defined]
+    elif t.scalarType() == "Float8_e4m3fn":
+        ret.tensor_type.elem_type = int(  # type: ignore
+            sym_hel._C_onnx.TensorProtoDataType.FLOAT8E4M3FN
```

Comment:
This should referred from onnx package instead for old torch
```suggestion
            onnx.TensorProto.DataType.FLOAT8E4M3FN  # type: ignore[attr-defined]
```

Reply:
Thank you. Fixed as such: 239ae63d85d3bbd76fe380cddaf07cf6f14d6b48